tests/client/amo/components/TestSearchResult.js

```diff
@@ -4,15 +4,21 @@ import {
   findRenderedDOMComponentWithClass,
   renderIntoDocument,
 } from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
 
+import createStore from 'amo/store';
 import SearchResult from 'amo/components/SearchResult';
 import { getFakeI18nInst } from 'tests/client/helpers';
 
 
 describe('<SearchResult />', () => {
   function renderResult(result) {
+    const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
     return findRenderedComponentWithType(renderIntoDocument(
-      <SearchResult i18n={getFakeI18nInst()} result={result} lang="en-US" />
+      <Provider store={createStore(initialState)}>
```

Comment:
I thought about doing that and will file an issue for it; it's a fair bit of boilerplate but I tried my hand at it really quickly and it needed some tweaking, and I tend to prefer writing easy app code over easy test code 😅